misc/scripts/deploy-tools.sh

```diff
@@ -60,6 +62,42 @@ function deployvm {
     done
 }
 
+function deployplugin {
+    for ip in $IP_LIST
+    do
+        TARGET=root@$ip
+        log "Cleaning up older files from $TARGET if exists..."
+        $SSH $TARGET rm -fr $TMP_LOC
```

Comment:
not a big deal, but it's better to run 1 ssh , not 3.
Something like `ssh $target "rm -fr $TMP_LOC;  $MKDIR_P $TMP_LOC $BUILD_LOC; "